tests/utils/test_plot.py

```diff
@@ -17,17 +17,28 @@ def test_check_list_type(self):
         self.assertTrue(check_list_type([12, 13, "text"], (int, str)))
 
     def test_add_x_to_plotting_options(self):
-        # test with empty dictionnary
+        # test without option_cat dictionary
         plotting_options = {}
-        self.assertDictEqual(add_x_to_plotting_options(plotting_options, 'colorbar', '#ee048e'),
-                             {'colorbar': '#ee048e'})
-
-        # test with not-empty dictionnary
-        plotting_options = {'log': True}
-        self.assertDictEqual(add_x_to_plotting_options(plotting_options, 'colorbar', '#ee048e'),
-                             {'log': True, 'colorbar': '#ee048e'})
-
-        # test with value already in plotting options
-        plotting_options = {'colorbar': 'red'}
-        self.assertDictEqual(add_x_to_plotting_options(plotting_options, 'colorbar', '#ee048e'),
-                             {'colorbar': 'red'})
+        self.assertDictEqual(add_x_to_plotting_options(plotting_options, 'traces', 'marker_color', '#ee048e'),
+                             {'traces': {'marker_color': '#ee048e'}})
+
+        # test with option_cat dictionary but without x
+        plotting_options = {'traces': {'marker_opacity': 0.5}}
+        self.assertDictEqual(add_x_to_plotting_options(plotting_options, 'traces', 'marker_color', '#ee048e'),
+                             {'traces': {'marker_opacity': 0.5, 'marker_color': '#ee048e'}})
+
+        # test with x already in plotting options[option_cat]
+        plotting_options = {'traces': {'marker_color': 'red'}}
+        self.assertDictEqual(add_x_to_plotting_options(plotting_options, 'traces', 'marker_color', '#ee048e'),
+                             {'traces': {'marker_color': 'red'}})
+
+    def test_add_default_titles_to_plotting_options(self):
+        title="graph about something"
+        xlabel="something"
+        ylabel="samples"
+
+        # test with empty dictionary
+        plotting_options = {}
+        print(add_default_titles_to_plotting_options(plotting_options, title, xlabel, ylabel))
```

Comment:
I think you forgot to remove a print statement here and replace it by the equality test between dictionnaries

Reply:
Yep, I saw it when I was correcting for flake8 ! :)